pkg/tempo/config.go

```diff
@@ -110,49 +117,74 @@ func (c *PushConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
-func (c *InstanceConfig) otelConfig() (*configmodels.Config, error) {
-	otelMapStructure := map[string]interface{}{}
+// DefaultRemoteWriteConfig holds the default settings for a PushConfig.
+var DefaultRemoteWriteConfig = RemoteWriteConfig{
+	Compression: compressionGzip,
+}
 
-	if len(c.Receivers) == 0 {
-		return nil, errors.New("must have at least one configured receiver")
+// RemoteWriteConfig controls the configuration of an exporter
+type RemoteWriteConfig struct {
+	Endpoint           string                 `yaml:"endpoint"`
+	Compression        string                 `yaml:"compression"`
+	Insecure           bool                   `yaml:"insecure"`
+	InsecureSkipVerify bool                   `yaml:"insecure_skip_verify"`
+	BasicAuth          *prom_config.BasicAuth `yaml:"basic_auth,omitempty"`
+	SendingQueue       map[string]interface{} `yaml:"sending_queue,omitempty"`    // https://github.com/open-telemetry/opentelemetry-collector/blob/7d7ae2eb34b5d387627875c498d7f43619f37ee3/exporter/exporterhelper/queued_retry.go#L30
+	RetryOnFailure     map[string]interface{} `yaml:"retry_on_failure,omitempty"` // https://github.com/open-telemetry/opentelemetry-collector/blob/7d7ae2eb34b5d387627875c498d7f43619f37ee3/exporter/exporterhelper/queued_retry.go#L54
+}
+
+// UnmarshalYAML implements yaml.Unmarshaler.
+func (c *RemoteWriteConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	*c = DefaultRemoteWriteConfig
+
+	type plain RemoteWriteConfig
+	if err := unmarshal((*plain)(c)); err != nil {
+		return err
+	}
+
+	if c.Compression != compressionGzip && c.Compression != compressionNone {
+		return fmt.Errorf("unsupported compression '%s', expected 'gzip' or 'none'", c.Compression)
```

Comment:
Something I learned about recently :) 
```suggestion
		return fmt.Errorf("unsupported compression %q, expected 'gzip' or 'none'", c.Compression)
```

Reply:
Actually, it would make the error look worse by mixing double quotes and single quotes. 🤷 Going to mark this as resolved.